master/buildbot/data/properties.py

```diff
@@ -15,6 +15,7 @@
 
 from buildbot.data import base
 from buildbot.data import types
+from twisted.internet import defer
```

Comment:
Can we somehow agree on imports? :)

I'd prefer to see imports from `twisted` before imports from `buildbot`...